modules/logging-set-output-rate-limit.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * logging/performance_reliability/logging-flow-control-mechanisms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-set-output-rate-limit_{context}"]
+= Configuring log forwarder output rate limits
+
+You can limit the rate of outbound logs to a specified output by configuring the `ClusterLogForwarder` custom resource (CR).
+
+.Prerequisites
+
+* You have installed the {clo}.
+* You have administrator permissions.
+
+.Procedure
+
+. Add a `maxRecordsPerSecond` limit value to the `ClusterLogForwarder` CR for a specified output.
++
+The following example shows how to configure an output rate limit for a Kafka broker output named `kafka-example`:
```

Comment:
Can we add something about per collector deployment here?  Your previous example explains the usage but I have concerns the message will not convey in this example and we'll get usage issues as a result.  Maybe?

... output rate limit per collector for a ...